blocks.py

```diff
@@ -18,4 +18,24 @@ def gyro_straight(tank, gyro, millies, target):
         subtract = gyro.angle() - target  # subtracts color sensors
         multiply = subtract * SHARPNESS # multiplies the difference by 0.7
         tank.drive(SPEED, multiply) # follows the line
+        
+        
+def stop_on_black(left_cs, right_cs):
+        if left_cs.reflection() <= 14:
+            return True
+        else:
+            return False
 
+        
+def perpendicular_line(tank, left_cs, right_cs, line_distance, left_wheel, right_wheel):
+    if line_distance >= 0:
+        turn = 85
+    else:
+        turn = -85
+    tank.settings(200, 100, 100, 100)
```

Comment:
We should probably call `tank.stop()` or document that it is expected for the robot to be in stop position when this function is called.

Reply:
I think it better to document this as calling stop wouldn't be doing anything, will get to fixing some of these outliers tomorrow morning